### ISSUE TITLE
Autoexpire “now playing” after song duration

### ISSUE THREAD

**Issue by MacLemon** [anlumo/KaraokeV2#45]
## Story or task
As a user I want to see if a song is currently playing or already through and there's probably moderation going on.

## Thoughts
- From the moment on when the PLAY button is pressed in the Queue, a timer should count down from song-duration until it reaches zero.
- Then the “now playing” marker is removed and the song is moved to history.
- Until the next song is playing, display a placeholder item “Moderation” between history and “up next”. (See #43 )
- Once Play is pressed, remove “Moderation” and advance “Up next” to “now playing”
- Visual representation could be a commonly know progress bar like item.

## Notes
- When the Play button is pressed we're likely already a few seconds in since this button is pressed when MC interaction with performers and Playback software has finished. (aka people are already singing.)
- guesstimating that we have about 10s delay between begin of a performance and the MC pressing play in the queue software.
- There is a rudimentary [API in Performous](https://github.com/performous/performous/wiki/Webserver-API)
  - there is no endpoint to get the currently playing song (as of 2024-05-08)
  - Playlist does not show currently playing song (since we cannot be operating in playlist mode)
  - at the moment we can't obtain real-time info from Performous to do better than an educated guess
  - Probably should request such an API endpoint with real-time info for Performous (and Ultrastar) 

## Accessibility note:
- Might help hearing impaired people to deduct if someone is singing or if we're in between songs. (People might not be in the same room, or actively watching the projector screen.)
- Might help visually impaired to check status properly when currently not in the same room.
